timetracker/timetracker/urls.py

```diff
@@ -17,17 +17,11 @@
 from django.contrib import admin
 from django.http import HttpResponse
 
-
-def hello_world(request):
-    name = request.GET.get('name')
-    if name:
-        return HttpResponse('Hello %s' % name)
-    else:
-        return HttpResponse('<form><input name="name"></form>')
-
-
+import entries.views
 
 urlpatterns = [
-    url(r'^$', hello_world),
+    url(r'^$', entries.views.home, name='homepage'), 
+	url(r'^clients/$', entries.views.clients, name='entries-viewClients'),
```

Comment:
Try to keep your indenting consistent -- use only spaces, or only tabs.

In general, everyone recommends 4-space indenting, _never_ tabs.  A space is always a space.  A tab may be shown as 2, 3, 4, 8, or any number of spaces, but Python will always consider it 8 spaces.  If you mix tabs and spaces, you may get "inconsistent indenting" errors and not understand why.